docs/adr/0000-odh-release-policy.md

```diff
@@ -0,0 +1,38 @@
+# Open Data Hub Release Policy 
+
+* Status: proposed
+* Deciders: vpavlin 
+* Date: 2020-Nov-03 
+
+Technical Story: As an Open Source projec, we want to document the policies and guideline on how we create a new
+release.
+
+## Context and Problem Statement
+
+The Open Data Hub itself consists of many components all having their own release cycles. ODH users might decide to
+updata individual components such as container images used by Jupyter Hub. Nevertheless it is required to create
+releases of ODH as a whole.
+
+## Considered Options
+
+* do a monolithic, coordinated release of all components of ODH by creating a tag within the odh-manifests repository
+* have a rolling release, and no tags on odh-manifests repository 
+
+## Decision Outcome
+
+Chosen option: we do a monolithic, coordinated release, because it will enable use to have a release at the
+project/product level while maintianing freedom of others to update.
+
+### Positive Consequences <!-- optional -->
+
+* Operators of ODH have a clear base line of versions, these versions have been tested with each other and have
+  undergone ODH integration testing.
+* Operators of ODH can update individual components, they could maintain a repository analoug to odh-manifests declaring
+  the exact versions (container image tags, git repository tags) of components they want to deploy.
+
+### Negative Consequences <!-- optional -->
+
+* A ODH release (a tag in the odh-manifests repository) might not contain the latest versions of components, for example
```

Comment:
Can we say this is also potentially resolved by deploying from master branch?

Reply:
but in that case the main branch will be a moving target, so that the operator deploys a pretty random set of component versions?!

Reply:
Ok, let's leave it this way for now...we can reword later:)

Reply:
I don't understand this? You mean vs a rolling update that always has the latest?

Reply:
ja